lua/chatcommands/coinflip.lua

```diff
@@ -0,0 +1,153 @@
+CoinFlips = CoinFlips or {}
+
+RegisterChatCommand({'coin','coinflip'},function(ply,arg)
+	local t = string.Explode(" ",arg)
+	local p = tonumber(table.remove(t))
+	if #t == 1 and t[1]:lower() == "accept" then checkCoinFlipRequest(ply, p) return end
+
+	if p == nil then
+		if #t == 1 and t[1]:lower() == "accept" then
+			ply:ChatPrint("[orange]!coinflip accept points")
+		else
+			ply:ChatPrint("[orange]!coinflip player points")
+		end
+	else
+		local to,c = PlyCount(string.Implode(" ",t))
+		if c == 1 then
+			if ply == to then
+				ply:ChatPrint("[red]You can't coinflip yourself!")
+			elseif hasCoinflipRequest(to) then
```

Comment:
Since you're forcing people to confirm the number of points, this is no longer necessary